nmostesting/suites/IS0401Test.py

```diff
@@ -861,7 +866,7 @@ def test_13(self, test):
                 if media_type not in supported_media_types:
                     if not warn_sdp_untested:
                         warn_sdp_untested = "Could not test Receiver {} because this test cannot generate SDP data " \
-                            "for media_type '{}'".format(receiver["id"], media_type)
+                            "for media_type '{}'".format(receiver[PRINT_KEY], media_type)

```

Comment:
get_resource_representation does not work here?

Reply:
It does, i just didn't need it, but with your augmentation change i'll just use it everywhere